docs/policies/policies.md

```diff
@@ -0,0 +1,515 @@
+        "odrl:action": {
+          "@id": "odrl:use"
+        },
+        "odrl:constraint": [
+          {
+            "@type": "odrl:LogicalConstraint",
+            "odrl:and": [
+              {
+                "odrl:leftOperand": {
+                  "@value": "<field>"
+                },
+                "odrl:operator": {
+                  "@id": "odrl:eq"
+                },
+                "odrl:rightOperand": "<value>"
+              }
+            ]
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+### Policies & Verifiable Credenitals (VC)
+
+#### General Information
+
+Catena-X uses policies to determine access to and use of data. The policies refer to verifiable credentials (VC) that are stored in the Managed Identity Wallet (MIW). Catena-X uses the principle of self-sovereign identity (SSI) here.
+
+ℹ️ Detailed Information about SSI, VC and MIW can be found here:
+
+- [https://github.com/eclipse-tractusx/ssi-docu](https://github.com/eclipse-tractusx/ssi-docu)
+- [https://github.com/eclipse-tractusx/ssi-docu/tree/main/docs/architecture/cx-3-2](https://github.com/eclipse-tractusx/ssi-docu/tree/main/docs/architecture/cx-3-2) (for release 23.09)
+- [https://github.com/eclipse-tractusx/ssi-docu/tree/main/docs/architecture/cx-3-3](https://github.com/eclipse-tractusx/ssi-docu/tree/main/docs/architecture/cx-3-3) (for release 23.12)
+
+The key architectural principle underlying this specification is that policy definitions must be decoupled from their corresponding VC schema. Namely, the specific **constraints** and shape of the VC schema must not be reflected in the policy definition. This allows VC schemas to be altered without impacting policy definitions.
+
+#### Membership Constraint Example
+
+The main difference between the various Catena-X policies is their `constraints`. For this reason, only the constraint examples are shown below instead of always showing the entire policy.
+
+To understand what the connection between policy and VC looks like, let's take a look at the following example ([Source](https://github.com/eclipse-tractusx/ssi-docu/blob/main/docs/architecture/cx-3-2/edc/policy.definitions.md#1-membership-constraint)):
+
+```json
+{
+    "constraint": {
+    "leftOperand": "Membership",
+    "operator": "eq",
+    "rightOperand": "active"
+    }
+}
+```
+
+An associated VC could look like this ([Source](https://github.com/eclipse-tractusx/ssi-docu/blob/main/docs/architecture/cx-3-2/3.%20Verifiable%20Credentials/Structure%20and%20Formats.md)):
+
+```json
+{
+    "id": "UUID",
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3id.org/security/suites/jws-2020/v1",
+        "https://raw.githubusercontent.com/catenax-ng/product-core-schemas/main/businessPartnerData"
+    ],
+    "type": ["VerifiableCredential", "MembershipCredentialCX"],
+    "issuanceDate": "2021-06-16T18:56:59Z",
+    "expirationDate": "2022-06-16T18:56:59Z",
+    "issuer": "did", // operating environment

```

Comment:
There are no comments in JSON.

Reply:
maybe having "comments" that invalidate the payloads will keep readers from just copying the examples ;)

Reply:
This example is 1:1 from the SSI repository (see source), but I will remove the comment ;)